system-test/Dockerfile.node12-alpine

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.12-alpine as builder
+RUN apk add --no-cache git
+WORKDIR /root/
+RUN go get github.com/google/pprof
+
+
+FROM node:12-alpine
+
+ARG ADDITIONAL_PACKAGES
+
+RUN apk add --no-cache bash $ADDITIONAL_PACKAGES
+WORKDIR /root/
+COPY --from=0 /go/bin/pprof /bin
```

Comment:
`--from=builder`

Reply:
Done (for this and other alpine docker files)